Header header
uint8 car_num
int32[] car_type
int32[] corner_point
float32[] distance